www/web/src/layouts/BaseLayout.astro

```diff
@@ -6,7 +6,7 @@ import Footer from '../components/Footer.astro'
 ---
 
 <html
-  class='scrollbar scrollbar-thumb-white scrollbar-track-[#1a172e] h-32 overflow-y-scroll'
+  class='scrollbar scrollbar-thumb-white scrollbar-track-[#1a172e] h-32 overflow-y-scroll dark'
```

Comment:
dark in itself is just a variant for tailwind, best to remove this as its not doing anything